src/components/ErrorDisplay.tsx

```diff
@@ -21,7 +21,7 @@ const ErrorDisplay = () => {
         },
       }}
     >
-      <Typography variant='body1'>Error: {error.message}</Typography>
+      <Typography variant='body1'>Error: {error?.message}</Typography>
```

Comment:
[req] it is not helpful to show `null` or `undefined` values in the error message. Additional condition with returning `null` could be added